### DISCUSSION TITLE
livecharts2 xaml set properties of axis (for example foreground, fontsize)

### DISCUSSION THREAD

**Discussion by atjbat** [Live-Charts/LiveCharts2#116] [General]
How I can do to set properties of axis (for example foreground, fontsize) in xaml?

**Comment by beto-rodriguez** [comment]
This is not possible for now, at least it is not as easy as it used to be in v0, you can try the [x:Array markup](https://docs.microsoft.com/en-us/dotnet/desktop/xaml-services/xarray-markup-extension#:~:text=x%3AArray%20is%20more%20intended%20for%20general%20language%20extensibility,an%20x%3AArray%20usage.%20x%3AArray%20is%20a%20markup%20extension.) but I am not a fan of it.

> Why?

Because it would be harder to maintain, we would require to build platform specific code for every XAML platform.
Maybe this feature could be added in a future.

> Alternatives? 

In V2 there are `Themes` you can customize the style of your charts globally, it is not documented properly yet, but here are the already built themes in the library:

[Light theme](https://github.com/beto-rodriguez/LiveCharts2/blob/31ca08dbd46aedfc7f7bbc90b48f09437b5c821e/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsSkiaSharp.cs#L81)

[Dark theme](https://github.com/beto-rodriguez/LiveCharts2/blob/31ca08dbd46aedfc7f7bbc90b48f09437b5c821e/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsSkiaSharp.cs#L207)

**Reply by atjbat** [reply]
ok tank you and compliments for your work
